components/ui/organisms/Header.tsx

```diff
@@ -114,7 +114,10 @@ export default function Header() {
                   </Link>
                 </li>
                 <li className="w-[100%]">
-                  <div className="w-[100%] mt-[1rem] text-center border-[1px] p-2 rounded-md border-primary font-normal">
+                  <div
+                    className="w-[100%] mt-[1rem] text-center border-[1px] p-2 rounded-md border-primary font-normal"
+                    style={{ border: '1px solid #0275d8' }}
```

Comment:
#00522A is the correct color, apply same change below.

Reply:
sure @CeoFred